src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.cmc.claimstore.documents;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.events.DocumentReadyToPrintEvent;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.sendletter.api.Letter;
+import uk.gov.hmcts.reform.sendletter.api.SendLetterApi;
+
+@Service
+@ConditionalOnProperty(prefix = "send-letter", name = "url")
+public class BulkPrintService {
+
+    private static final String XEROX_TYPE_PARAMETER = "CMC001";
```

Comment:
Would be nice to have a comment here which explains what this is and why it has this value.